core/vm-common/codeMetadata.go

```diff
@@ -0,0 +1,46 @@
+package vmcommon
+
+const lengthOfCodeMetadata = 2
+
+const (
+	METADATA_UPGRADEABLE = 1
```

Comment:
Please rename these to match with the rest of the project: MetadataUpgradeable....

Reply:
it was copy paste from vmcommon. not necessarily to update everywhere.

Reply:
done